### PR TITLE
Cleanup ephemeral

### PR DESCRIPTION
**Pull request by luckysori**
Trying to align setup actors with the rest of the ephemeral actors. Let me know if there's anything else.

### CODE REVIEW

File: daemon/src/address_map.rs [hunk 1]
```diff
@@ -61,6 +61,32 @@ where
             _ => Err(NotConnected::new::<A>()),
         }
     }
+
+    /// Sends a message to the actor stored with the given key.
+    pub async fn send_fallible<M>(
+        &self,
+        key: &K,
+        msg: M,
+    ) -> Result<anyhow::Result<()>, NotConnected>
```
Comment (thomaseizinger) [inline daemon/src/address_map.rs]: Do we need to qualify here? Don't think so :)

File: daemon/src/address_map.rs [hunk 2]
```diff
@@ -61,6 +61,32 @@ where
...
+            Some(addr) if addr.is_connected() => {
+                if let Err(e) = addr
+                    .send(msg)
+                    .await
+                    .expect("we checked that we are connected")
+                {
+                    return Ok(Err(e));
+                };
+
+                Ok(Ok(()))
```
Comment (thomaseizinger) [inline daemon/src/address_map.rs]: Isn't this equivalent:

```suggestion
                let result = addr.send(msg).await.expect("we checked that we are connected");
                Ok(result)
```

**Review by thomaseizinger — APPROVED** [review]

File: daemon/src/address_map.rs (hunk 1 above)
Comment (luckysori) [reply daemon/src/address_map.rs]: We don't if we import `anyhow::Result`!

File: daemon/src/address_map.rs (hunk 2 above)
Comment (luckysori) [reply daemon/src/address_map.rs]: Equivalent and better.

**Review by thomaseizinger — APPROVED** [review]

**Pull request merged**